Admin/AbstractAdmin.php

```diff
@@ -2845,6 +2852,22 @@ public function getDashboardActions()
     /**
      * {@inheritdoc}
      */
+    public function getSearchResultLink($object)
```

Comment:
This is not a reason. Enable overriding everywhere is why it's hard to have a flexible package without breaking things.

In this, I see no good reason to extends the `getSearchResultLink`.

If somebody wants to override it, it will be free to propose a PR and explain why he wants to do it and maybe we can find another solution for him.

cc @greg0ire